component_text/absrel.md

```diff
@@ -0,0 +1,97 @@
+# aBSREL
```

Comment:
It's all the basic text to go anywhere in the site. I don't understand the question.